perllib/Integrations/Bartec.pm

```diff
@@ -212,7 +212,7 @@ sub Jobs_FeatureScheduleDates_Get {
     my $days_buffer = 15;
 
     $start = $w3c->format_datetime($start || DateTime->now->subtract(days => $days_buffer));
-    $end = $w3c->format_datetime($end || DateTime->now);
+    $end = $w3c->format_datetime($end || DateTime->now->add(days => $days_buffer));
```

Comment:
The commit message probably needs updating to explain this change too. Basically we need to search in the future too because of [an apparent Bartec bug](https://3.basecamp.com/4020879/buckets/19439943/todos/4365548265).